utils/update_checkout/update-checkout-config.json

```diff
@@ -123,7 +125,7 @@
                 "swift-crypto": "3.0.0",
                 "swift-certificates": "1.0.1",
                 "swift-asn1": "1.0.0",
-                "swift-async-algorithms": "1.0.0",
+                "swift-async-algorithms": "main",
```

Comment:
Is there a reason we are using main branch now?

Reply:
We need to use a revision including https://github.com/apple/swift-async-algorithms/commit/6ae9a051f76b81cc668305ceed5b0e0a7fd93d20 to make SDK generator build work.
But it's not ideal situation, so I asked a new release including the fix just now https://github.com/apple/swift-async-algorithms/issues/314